src/rpc/rawtransaction.cpp

```diff
@@ -1081,6 +1081,43 @@ static RPCHelpMan decodepsbt()
                                 {
                                     {RPCResult::Type::STR_HEX, "", "hex-encoded witness data (if any)"},
                                 }},
+                                {RPCResult::Type::STR_HEX, "taproot_key_path_sig", /* optional */ true, "hex-encoded signature for the Taproot key path spend"},
```

Comment:
48d5949aa94410b12263df9b7a3cbc4d7cd50b6f: having a fully decoded example in a functional test case would be handy

Reply:
Is there not? There are tests added which check for the presence of the taproot fields in psbts spending taproot inputs.

Reply:
I noticed the presence checking tests, though perhaps checking against the fully decoded result could reveal more subtle issues.